modules/update-upgrading-web.adoc

```diff
@@ -5,7 +5,8 @@
 [id="update-upgrading-web_{context}"]
 = Updating a cluster by using the web console
 
-If updates are available, you can update your cluster from the web console.
+If updates are available, you can update your cluster from the web console. You
+can perform both major and minor updates from the web console.
```

Comment:
Major (e.g. 4.9.10->5.0.0), minor (e.g. 4.1.9->4.2.3), and patch (e.g. 4.1.9->4.1.11)?

Reply:
And in case we are disagreeing on terms, my major.minor.patch terms are from https://semver.org/spec/v2.0.0.html#summary

Reply:
Your terminology is better than mine.